documentation/book/proc-deploying-kafka-cluster-kubernetes.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-cluster.adoc
+
+[id='deploying-kafka-cluster-kubernetes-{context}']
+= Deploying the Kafka cluster to {KubernetesName}
+
+.Prerequisites
+
+* Before deploying a Kafka cluster, the Cluster Operator must be deployed.
+* To deploy a Kafka cluster, a `Kafka` resource with the cluster configuration has to be created.
```

Comment:
This is already the procedure, not prerequisite. This is what the `kubectl apply ...` command does.